src/app/core-ui/main/main-view.component.html

```diff
@@ -238,15 +238,14 @@
         {{ title }}
       </div>
     </div><!-- .left -->
-
     <div class="right" fxFlex="1 1 60%">
       <!-- FIXME: show only when stuff is in cart -->
-      <div class="cart-info TO_DO" routerLinkActive="active"
+      <div class="cart-info" routerLinkActive="active"
```

Comment:
Can we move this code into a separate component?
like the status bar basically..

https://github.com/vikas-cis/partgui/tree/ab00a2e2fcd5b2c2909930dcedb6b5b497c25fe9/src/app/core-ui/main

Reply:
also what's the routerLinkActive doing here, curious..

Reply:
lol I think its doing nothing we can remove it 👍